### ISSUE TITLE
Infinite loop in LHCO reader

### ISSUE THREAD

**Issue by jackaraz** [MadAnalysis/madanalysis5#136]
### System Settings

* OS: macOS v12.6
* Python 3.9.10
* Apple clang version 14.0.0 (clang-1400.0.29.102)
* MA5 release : 2.0.4 [ 2022/07/18 ]

### Describe the bug

Normal mode gets stuck in an infinite loop while reading an LHCO file.

### To Reproduce

```
./bin/ma5 -R
install samples
import samples/mg5_ttbar2l.lhco
plot PT(j)
submit
```
This will get stuck in the first event. if one writes a print function in 

https://github.com/MadAnalysis/madanalysis5/blob/a4a9edd088aed61186554686bde433575e278976/tools/SampleAnalyzer/Process/Reader/LHCOReader.cpp#L343-L369

one will see that it will read only 5 jets. and the analysis can be modified to see `INFO << "jet size " << event.rec()->jets().size() << endmsg;` which prints `jet size 7595718147998050672`.

### Expected behaviour

The number of jets in the reader and analysis should match.

**Comment by jackaraz** [comment]
*Update*
If one adds `event.rec()->Print();` in `user.cpp` the printout is as follows
```
   -------------------------------------------
   Event Content :
      * Jet Content :
         - Jet ID = Ma5Jet, Number of Jets = 7595718147998050672 (Primary)
      * Number of Taus      = 10013946782870899450
      * Number of Electrons = 0
      * Number of Muons     = 1
      * Number of Photons   = 0
   -------------------------------------------
```

**Comment by BFuks** [comment]
Some update. 

If you type:
```export FASTJET_FLAG="-DMA5_FASTJET_MODE"```
before starting MA5, it works. I will now investigate this further.

Cheers!

**Comment by jackaraz** [comment]
This particular issue has been resolved in PR #167
